apps/discord-bot/cmd/main/main.go

```diff
@@ -73,6 +73,8 @@ func main() {
 		}
 	}
 
+	command := command.NewCommands(discordClient, token, service, 2*time.Second)
```

Comment:
We possess a package named command. Whenever we wish to create a new command, we can utilize command.New(discordClient, token, service, 2*time.Second). Consequently, it would be more appropriate to rename the function NewCommands to New.